WordPress/wordpress-playground
# Blueprints: Resolve the latest WordPress version from the API instead of assuming it's the same as the last minified build

https://github.com/WordPress/wordpress-playground/pull/1987 created a dependency between the `@wp-playground/blueprints` package and  the `@wp-playground/wordpress-builds` package breaking `@wp-playground/cli` – see https://github.com/WordPress/wordpress-playground/issues/2026

This PR updates the setSiteLanguage step to pull the latest/best WordPress version details from api.wordpress.org/core/version-check/1.7?channel=beta instead of implicitly assuming the latest version is the same as that of the latest minified web build. It reuses the Playground CLI resolveWordPressRelease function, bringing us closer to having zero CLI-specific logic.

## Follow-up work

Invent an ESLint rule to prevent further dependencies on `@wp-playground/wordpress-builds`

 ## Testing instructions

* CI tests
* Run Playground CLI via `bun packages/playground/cli/src/cli.ts server` and confirm WordPress is still being downloaded without errors.

cc @swissspidy @bgrgicak
